### PR TITLE
AccordionItem and ExpandAble preparation for css modules transition

### PR DESCRIPTION
**Pull request by orrgottlieb**
* Adding classnames to customize specific place in Expanable
* Improve CSS structure for Expandable

### CODE REVIEW

**Review by SergeyRoyt — APPROVED** [review]

File: src/components/ExpandCollapse/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.hideBorder {
```
Comment (SergeyRoyt) [inline src/components/ExpandCollapse/styles.module.scss:1]: Maybe to rename the file to `ExpandCollapse.module.scss` or smth? Just to keep the convention

File: src/components/ExpandCollapse/ExpandCollapse.tsx [hunk 2]
```diff
@@ -7,12 +7,15 @@ import { VibeComponentProps } from "../../types";
 import cx from "classnames";
 import { ElementContent } from "../../types/ElementContent";
 import "./ExpandCollapse.scss";
+import extraStyles from "./styles.module.scss";
```
Comment (SergeyRoyt) [inline src/components/ExpandCollapse/ExpandCollapse.tsx:10]: Maybe to rename the file to`ExpandCollapse.module.scss` or smth? Just to keep the convention

Comment (SergeyRoyt) [reply src/components/ExpandCollapse/ExpandCollapse.tsx:10]: Or maybe it's better to leave it this way - would be less confusing when I'll merge with Vibe 2.0

Comment (orrgottlieb) [reply src/components/ExpandCollapse/ExpandCollapse.tsx:10]: thought about 2.0 so afterwards we would just merge the both

**Pull request merged**